tests/e2e/cluster_downgrade_test.go

```diff
@@ -135,7 +171,8 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int, triggerSnapshot bool) {
 	beforeMembers, beforeKV = getMembersAndKeys(t, cc)
 
 	t.Logf("Starting upgrade process to %q", currentVersionStr)
-	e2e.DowngradeUpgradeMembers(t, nil, epc, len(epc.Procs), lastClusterVersion, currentVersion)
+	err = e2e.DowngradeUpgradeMembers(t, nil, epc, len(epc.Procs), lastClusterVersion, currentVersion)
```

Comment:
Any plans to test cancellation mid downgrade?

Reply:
Yes! In the draft Pr (part 2)

There is a potential bug that I am investigating (discussed with @siyuanfoundation)

Reply:
SG